build/Targets/VSL.Versions.targets

```diff
@@ -9,6 +9,7 @@
     <SystemCollectionsImmutableVersion>1.2.0-rc3-24102-00</SystemCollectionsImmutableVersion>
     <MicrosoftDiaSymReaderVersion>1.0.8-rc3-60429-03</MicrosoftDiaSymReaderVersion>
     <MicrosoftDiaSymReaderNativeVersion>1.4.0-rc2</MicrosoftDiaSymReaderNativeVersion>
+    <MicrosoftDiaSymReaderPortablePdbVersion>1.0.0</MicrosoftDiaSymReaderPortablePdbVersion>
```

Comment:
No , lets wait for @tmat take a look at this and the project.json version changes and he can send another PR with necessary changes. This should unblock you for now.